src/codeu/chat/client/commandline/Chat.java

```diff
@@ -119,7 +131,7 @@ public void invoke(Scanner args) {
     //
     panel.register("u-list", new Panel.Command() {
       @Override
-      public void invoke(Scanner args) {
```

Comment:
For this code, The parameter type is still "Scanner" in my code not List<String>.
We should come up with a same type.